packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposeConfig.ts

```diff
@@ -66,6 +75,30 @@ const validateInput = (input: CommandInput): boolean => {
   return true
 }
 
+const beforeExecute: BeforeExecute<CommandInput, ContractInput> = (context) => async () => {
+  const event = await getLatestOCRConfig(context.provider, context.contract)
+
+  const contractConfig = {
+    f: event?.f[0],
+    transmitters: event?.transmitters,
+    signers: event?.signers.map((s) => Buffer.from(s, 'hex').toString('base64')),
+    onchain_config: event?.onchain_config[0],
+    // todo: add payees
```

Comment:
The payees cant be fetched from the contract? Please add more information about this TODO in form of inline docs.

Reply:
Yeah, the contract doesn't provide payees getter nor payees are returned in the log event. Rodrigo opened an issue somewhere for that, but I just in case I created another one here - https://github.com/smartcontractkit/chainlink-terra/issues/180